### DISCUSSION TITLE
Feature : team management

### DISCUSSION THREAD

**Discussion by bynicodevelop** [verdaccio/verdaccio#3076] [Ideas]
Hello,

Is it planned in future versions to support team management?

Thank you in advance for your response.

**Comment by juanpicado** [comment]
hi @bynicodevelop , not on the roadmap, the https://docs.npmjs.com/cli/v7/commands/npm-team is so deeply integrated into npmjs business logic and completely closed that unique alternative would be "made up" something based on what you see on the npm CLI. They can change easily the logic in future major releases and break the registry, it is hard to follow up if is a closed source. I haven't pursued personally this command for that and other lack of time reasons. 

Not on my plans in middle-term, but ideas are welcome.
